src/addressBook/index.js

```diff
@@ -141,21 +141,14 @@ export async function getPeople(containerIri, fetch) {
   const profiles = profileResponses
     .filter(({ error: e }) => !e)
     .map(({ response }) => response)
+    .filter(({ iri }) => iri.includes("/profile/card#me")) // filtering by valid WebId so it doesn't error out if the user entered a contact with an invalid ID
```

Comment:
WebIds are not necessarily following the form `/profile/card#me`, that's just a form that NSS uses.

(If anything, we need to resolve "bad" webIds when we fetch them. But we should create another ticket to handle those situations.)